moveit_setup_assistant/resources/default_config/sensors_3d.yaml

```diff
@@ -0,0 +1,22 @@
+# This YAML configuration file holds the default values used for configuring 3D sensors.
+# The name of this file shouldn't be changed, or else the Setup Assistant won't detect it.
+# Values may not be ideal defaults, original source unclear.
+sensors:
+  - sensor_plugin: occupancy_map_monitor/PointCloudOctomapUpdater
+    point_cloud_topic: /head_mount_kinect/depth_registered/points
+    max_range: 5.0
+    point_subsample: 1
+    padding_offset: 0.1
+    padding_scale: 1.0
+    max_update_rate: 1.0
+    filtered_cloud_topic: filtered_cloud
+  - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
```

Comment:
Won't have both of them in the same file end up loading both of them?

Reply:
Yes, but we don't put this file on the parameter server, we only use it for holding the default values.